src/common/Backend.ts

```diff
@@ -133,6 +134,27 @@ export class ServerError extends Error {
         return new ServerError(json.message, json.description, json.status);
     }
 }
+export class CancelError extends Error {
+    type = 'cancel';
+
+    message: string;
+
+    config: AxiosRequestConfig | undefined;
+
+    constructor(message: string | undefined, config?: AxiosRequestConfig) {
+        super();
+        this.message = message ?? 'The request was cancelled';
+        this.config = config;
+    }
+
+    static isCancel(cancel: unknown): cancel is CancelError {
+        return cancel instanceof CancelError;
+    }
```

Comment:
Please remove this method. `ServerError` only has `isJson` to detect whether some JSON sent by the server is a server error. Obviously, we don't need to do something like that for cancellation.